spec/config/mongoid.rb

```diff
@@ -2,12 +2,14 @@
   config.connect_to('delayed_job_shallow_mongoid_test')
 end
 
+Mongo::Logger.logger = ::Logger.new('tmp/mongo.log') if defined?(Mongo)
+
 RSpec.configure do |config|
   config.before(:each) do
     Mongoid.purge!
     Mongoid::IdentityMap.clear if Delayed::ShallowMongoid.mongoid3?
   end
   config.after(:all) do
-    Mongoid.default_session.drop
+    Mongoid.default_session.drop if Mongoid.respond_to?(:default_session)
```

Comment:
I was getting a `NoMethodError: undefined method `default_session' for Mongoid:Module` so I just simply ignored it when it's not dafined. But there must be a better way to terminate the connection properly rather than just skipping it.